mmcv/ops/csrc/pytorch/pybind.cpp

```diff
@@ -312,9 +312,9 @@ Tensor nms_rotated(const Tensor dets, const Tensor scores, const Tensor order,
                    const Tensor dets_sorted, const float iou_threshold,
                    const int multi_label);
 
-Tensor upfirdn2d(const Tensor &input, const Tensor &kernel, int up_x, int up_y,
-                 int down_x, int down_y, int pad_x0, int pad_x1, int pad_y0,
-                 int pad_y1);
+Tensor upfirdn2d(torch::Tensor x, torch::Tensor f, int upx, int upy, int downx,
```

Comment:
Is it necessary to add `torch::`?

Reply:
The name of parameters `x` and `f` are too short.

Reply:
x -> input, f -> filter